ui/partials/perf/compareconfidence.html

```diff
@@ -1,3 +1,8 @@
-<span ng-if="text && value">
-  {{value|displayPrecision}} (<abbr title="Result of running t-test on base versus new result distribution: higher values indicate greater confidence that the indicated performance difference is statistically meaningful">{{text}}</abbr>)
+<span ng-if="result.confidenceText && result.confidence">
+  {{result.confidence|displayPrecision}} (<abbr title="Result of running t-test on base versus new result distribution: higher values indicate greater confidence that the indicated performance difference is statistically meaningful">{{result.confidenceText}}</abbr>)
+  <span ng-if="result.originalRuns.length < 5 || result.newRuns.length < 5"
+        class="glyphicon glyphicon-warning-sign text-warning"
+        tooltip="Need at least 5 runs for each push (base and new) to have full confidence in the comparison"
```

Comment:
Testing locally. Perhaps there is a way to insulate the tooltip from the `.subtest-highlighted` class applied to the row? For lines whose confidence value is bold, the tooltip seems to become bold also.

![boldtooltip](https://cloud.githubusercontent.com/assets/3660661/8726248/09a72e9e-2ba7-11e5-9ea2-33c8c26bd8e7.jpg)